book.rb

```diff
@@ -0,0 +1,10 @@
+require './rental'
+class Book
+  attr_accessor :title, :author
```

Comment:
Hey Thank you for your review, with adding :rentals to attr_accessor, wouldn't it make more sense to add attr_reader, since we wouldn't want to modify the rentals?